osu.Game/Rulesets/Judgements/JudgementPiece.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Extensions;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Rulesets.Scoring;
+
+namespace osu.Game.Rulesets.Judgements
+{
+    public abstract partial class JudgementPiece : CompositeDrawable
+    {
+        protected readonly HitResult Result;
+
+        protected SpriteText JudgementText { get; set; } = null!;
+
+        protected JudgementPiece(HitResult result)
+        {
+            Result = result;
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            JudgementText.Text = Result.GetDescription().ToUpperInvariant();
```

Comment:
This is a bit dodgy... Basically all implementors of this are expected to know that they should (a) create `JudgementText` themselves, and (b) do so before `base.LoadComplete()`. Is this needed? Can we expose just the `LocalisableString` with the text or something? Or move the creation of the judgement text to an abstract method that you can't not implement?

Reply:
I had more in that direction on a separate branch, will move it over here.